.gitignore

```diff
@@ -1,4 +1,10 @@
 
 .DS_Store
 .vscode/
-build/
\ No newline at end of file
+build/
+VS/.vs/cudaDeconv/v15/Solution.VC.db
```

Comment:
not sure if there's stuff in `VS/.vs/` that you _don't_ want to ignore, but if you just want to ignore the folder these lines can just be `VS/.vs/cudaDeconv/v15/`.